src/elements/content-sidebar/stories/tests/BoxAISidebar-visual.stories.tsx

```diff
@@ -25,11 +55,34 @@ export const BoxAIInSidebar: StoryObj<typeof BoxAISidebar> = {
     },
 };
 
-export const BoxAIWithExpandedView: StoryObj<typeof BoxAISidebar> = {
+export const BoxAIWithAgentSelectorOpened: StoryObj<typeof BoxAISidebar> = {
+    args: {
+        features: {
+            ...mockFeatures,
+            'boxai.agentSelector.enabled': true,
+        },
+        boxAISidebarProps: {
+            onClearClick: fn,
+            agents: mockAgents,
+            selectedAgent: mockAgents[0],
+        },
+
+    },
+    play: async () => {
+        await waitFor(async () => {
+            const agentSelector = await screen.findByRole('button', {name: 'Agent Agent 1'}, { timeout: 5000 });
```

Comment:
do we need to set it to 5000? do we need it at all? youre already doing a waitFor and a findByRole, that should be enough to avoid waiting 5000